backend/docker-compos.e2e.yml

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: 2025 The HedgeDoc developers (see AUTHORS file)
+# SPDX-License-Identifier: AGPL-3.0-only
+---
+services:
+  mariadb:
+    # renovate: datasource=docker depName=mariadb
+    image: mariadb:11.5.2@sha256:2d50fe0f77dac919396091e527e5e148a9de690e58f32875f113bef6506a17f5
+    environment:
+      MYSQL_USER: hedgedoc
+      MYSQL_PASSWORD: hedgedoc
+      MYSQL_DATABASE: hedgedoc
```

Comment:
The official docs for the mariadb image use `MARIADB_` variables instead of `MYSQL_` ones.
While there's still backward-compatibility, it might be removed at some point of time and we should probably use the up-to-date prefix therefore.

See: https://hub.docker.com/_/mariadb#-via-docker-compose